outwatch/src/main/scala/outwatch/interpreter/SnabbdomOps.scala

```diff
@@ -117,7 +117,7 @@ private[outwatch] object SnabbdomOps {
 
       var proxy: VNodeProxy = null
       var nextModifiers: js.UndefOr[js.Array[StaticVDomModifier]] = js.undefined
-      var prependModifiers: js.UndefOr[js.Array[StaticVDomModifier]] = js.undefined
+      var _prependModifiers: js.UndefOr[js.Array[StaticVDomModifier]] = js.undefined
```

Comment:
The naming of either the variable or the parameters is poor. Which should be renamed?

Reply:
Why do we need to rename them?

Reply:
I do not know why they cannot have the same name as the call-by-name params, but it would not let me compile if they had the same name.

Reply:
You mean the named arguments? I also do not understand. Will leave it like this for now then. I will try to find out what is happening :)